Sources/FigmaExport/Loaders/ImagesLoader.swift

```diff
@@ -130,27 +130,52 @@ final class ImagesLoader {
         }
         
         let imagesIds: [NodeId] = imagesDict.keys.map { $0 }
+        let scales = platform == .android ? [1, 2, 3, 1.5, 4.0] : [1, 2, 3]
 
         var images: [Double: [NodeId: ImagePath]] = [:]
-        images[1.0] = try loadImages(fileId: fileId, nodeIds: imagesIds, params: PNGParams(scale: 1.0))
-        images[2.0] = try loadImages(fileId: fileId, nodeIds: imagesIds, params: PNGParams(scale: 2.0))
-        images[3.0] = try loadImages(fileId: fileId, nodeIds: imagesIds, params: PNGParams(scale: 3.0))
-        if platform == .android {
-            images[1.5] = try loadImages(fileId: fileId, nodeIds: imagesIds, params: PNGParams(scale: 1.5))
-            images[4.0] = try loadImages(fileId: fileId, nodeIds: imagesIds, params: PNGParams(scale: 4.0))
+        for scale in scales {
+            images[scale] = try loadImages(fileId: fileId, nodeIds: imagesIds, params: PNGParams(scale: scale))
         }
-        return imagesIds.map { imageId -> ImagePack in
-            let name = imagesDict[imageId]!.name
 
-            var scaledImages: [Double: Image] = [:]
-                
-            images.forEach { scale, idToPath in
-                let url = URL(string: idToPath[imageId]!)!
-                let image = Image(name: name, url: url, format: "png")
-                scaledImages[scale] = image
+        /*
+         # imagesDict
```

Comment:
should we remove the swift comments? 🤓

Reply:
I would like to keep it, so others can quick understand the data conversion.